Ghidra/Features/GraphServices/src/main/resources/jungrapht.properties

```diff
@@ -27,7 +27,7 @@ jungrapht.edgeWidth=4.0f
 # stroke size for the magnifier lens
 jungrapht.lensStrokeWidth=10.0
 # when scale is < .3, switch to lightweight rendering
-jungrapht.lightweightScaleThreshold=.3
+jungrapht.lightweightScaleThreshold=.8
```

Comment:
cross over to simple view 'sooner' to avoid un-readable vertex labels (when scaled too small)